pkg/processor/test/suite/suite.go

```diff
@@ -87,6 +101,80 @@ func (suite *TestSuite) SetupTest() {
 	suite.TestID = xid.New().String()
 }
 
+// BlastHTTP is a stress test suite
+func (suite *TestSuite) BlastHTTP(request StressRequest) bool {
```

Comment:
Try to make your functions read like stories - where the reader doesn't have to get buried in details to get the big picture. This means short functions that call other functions that do one thing. In this case, `BlastHTTP` can simply call:

```
deployOptions, err := suite.blastConfigurationToDeployOptions(&blastConfiguration)
<check>

_, err := suite.Platform.DeployFunction(deployOptions)
<check>

totalResults, err := suite.blastFunction(&blastConfiguration)
<check>

suite.Platform.DeleteFunction...
<check>

<assert results>
```

Reply:
If there is no way I could return error from a function in current circumstances - is it better to still make error one of the return values (for later changes)?